src/discordcr/rest.cr

```diff
@@ -779,9 +779,9 @@ module Discord
     # "Manage Channels" permission.
     #
     # [API docs for this method](https://discordapp.com/developers/docs/resources/guild#modify-guild-channel-positions)
-    def modify_guild_channel_positions(guild_id : UInt64,
+    def modify_guild_channel_positions(guild_id : UInt64 | Snowflake,
                                        positions : Array(ModifyChannelPositionPayload))
-      request(
+      response = request(
```

Comment:
??

Reply:
Ah yeah, sorry about these/others. Must have missed them in the rebase.